scripts/release.sh

```diff
@@ -164,7 +199,12 @@ if [[ $? == 1 ]]; then
 
     # This runs Jazzy to generate the documentation.
     echo "Running Jazzy to generate documentation..."
-    jazzy --clean --objc --framework-root SmartDeviceLink --sdk iphonesimulator --umbrella-header SmartDeviceLink/public/SmartDeviceLink.h --theme theme --output docs
+    # TODO - I do not know if this will work as intended.
+    if [ $(uname -m) == "x86_64" ]; then
+        jazzy --clean --objc --framework-root SmartDeviceLink --sdk iphonesimulator --umbrella-header SmartDeviceLink/public/SmartDeviceLink.h --theme theme --output docs
+    else
+        arch -x86_64 jazzy --clean --objc --framework-root SmartDeviceLink --sdk iphonesimulator --umbrella-header SmartDeviceLink/public/SmartDeviceLink.h --theme theme --output docs
+    fi
```

Comment:
Should this call `generate_documentation.sh` and can this fix be applied there?

Reply:
I think we were at one time, and then we decided to copy the contents into the release script.  Then we decided to keep the generate script jsut in case we needed it.

I am testing calling it now and I may need to fix it before we can call it.

Reply:
I've got that fixed.